### DISCUSSION TITLE
Kills not counted in Sub Vs Sub Combat

### DISCUSSION THREAD

**Discussion by 4-S-H** [FakeFishGames/Barotrauma#14918] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Kills not counted in Sub Vs Sub Combat

### Reproduction steps

- Start a Sub vs Sub PVP round
- Shoot someone until they fall unconscious
- They die
- Kill and death not counted at the end

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
Kills and deaths are counted correctly in PVP game modes.
Closing discussion as resolved.
